README.md

```diff
@@ -20,7 +20,7 @@ To be able to run an installation from the repo (and not from a pre-built packag
 
 1. [Create an account on phpBB.com](http://www.phpbb.com/community/ucp.php?mode=register)
 2. [Create a ticket (unless there already is one)](http://tracker.phpbb.com/secure/CreateIssue!default.jspa)
-3. Read our [Coding guidelines](https://wiki.phpbb.com/Coding_guidelines) and [Git Contribution Guidelines](http://wiki.phpbb.com/Git)
+3. Read our [Coding guidelines](https://wiki.phpbb.com/Coding_guidelines) and [Git Contribution Guidelines](https://area51.phpbb.com/docs/dev/32x/development/git.html)
```

Comment:
coding guidelines should be https://area51.phpbb.com/docs/dev/development/coding_guidelines.html
git guidelines should be https://area51.phpbb.com/docs/dev/development/git.html